docs/README.md

```diff
@@ -12,6 +12,27 @@ To contact us you can:
 
 The source code is available [on GitHub](https://github.com/mozilla/glean/).
 
-## License
+## Using this book
 
-This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at <https://mozilla.org/MPL/2.0/>.
```

Comment:
Shouldn't we retain the license to Glean somewhere on the README?  (I know it doesn't really pertain to the docs, but I think it's probably still an important detail about the project)

Reply:
(But totally fine with moving it to the bottom / de-emphasizing it etc.)